buildconfig/Jenkins/Conda/download-and-install-mambaforge

```diff
@@ -39,6 +44,14 @@ if [[ ! -f $EXPECTED_CONDA_PATH ]]; then
             exit 1
         fi
     fi
+
+    # Compare the sha256 of the downloaded file to the expected sha256
+    SHA256_DOWNLOADED=$(sha256sum $MAMBAFORGE_SCRIPT_NAME)
```

Comment:
Not sure how we have the build servers set up, but are our windows machines going to have `sha256sum` available to them?

Reply:
They will as it is shipped as part of Git Bash (GNU coreutils) which is what we are utilizing for these scripts on windows.

Reply:
Do the mac builders have it installed too? It doesn't seem to be there by default, at least not on my local machine

Reply:
Just checked on the builders and they don't so I changed the code, great catch!